test/unit/fast_math/fast_math.cpp

```diff
@@ -96,6 +96,20 @@ SCENARIO("Check fast_math") {
     GIVEN("exprelr (float)") {
         auto test = check_over_span(exprelr_ref, exprelr, low_limit_f, high_limit_f, npoints);
 
+        THEN("error inside threshold") {
+            REQUIRE(test);
+        }
+    }
+    GIVEN("log10 (double)") {
+        auto test = check_over_span(std::log10, log10, low_limit, high_limit, npoints);
```

Comment:
the limits should be different for `log10`, I believe the range should ~ ]std::numeric_limits<T>::min(), std::numeric_limits<T>::max()[